test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/vars.tf

```diff
@@ -14,6 +14,20 @@ variable "CI_BUILD_NUMBER" {
   default = "unknown-build"
 }
 
+variable "BRANCH_NAME" {
+  description = "Branch name for tagging purposes"
+  default = "unknown-branch"
+}
+
+variable "BUILD_ID" {
```

Comment:
We have `CI_BUILD_NUMBER` above, which seems to be the same as `BUILD_ID`. I think we can remove one of them.

Reply:
Indeed, I missed those somehow , good catch